pkgs/tools/typesetting/asciidoc/default.nix

```diff
@@ -262,6 +262,7 @@ stdenv.mkDerivation rec {
     patchShebangs .
 
     sed -i -e "s,/etc/vim,,g" Makefile.in
+    sed -i -e "s,python3 a2x.py,python3 a2x.py -a revdate=01/01/1980,g" Makefile.in
```

Comment:
Looks ok to me in principle, but it seems that this will break in the next release:

https://github.com/asciidoc/asciidoc-py3/blob/e76294bc6b1fe878a2ed5d5d4816385d600574da/Makefile.in#L131

Maybe it's worthwhile using `substituteInPlace` here, since it will give a warning when the substitution does not make any changes? Since we know that this has be changed in the next release, maybe it's also a good to put a comment in the `src` attribute warning about this?

Reply:
@danieldk something like this? https://github.com/tomberek/nixpkgs/commit/fac4450a9d475a49375f89a8c4b2864f931f0da9

Reply:
> @danieldk something like this? [tomberek@fac4450](https://github.com/tomberek/nixpkgs/commit/fac4450a9d475a49375f89a8c4b2864f931f0da9)

It's still a bit hard to read without context. Maybe:

```
# Note: this substitution will not work in the planned 10.0.0 release:
#
# https://github.com/asciidoc/asciidoc-py3/commit/dfffda23381014481cd13e8e9d8f131e1f93f08a
#
# Update this substitution to:
#
# --replace "python3 -m asciidoc.a2x" "python3 -m asciidoc.a2x -a revdate=01/01/1980"
```

Though it would also be good to add something above the

```
src = fetchFromGitHub {
```

line like:

```
# Note: a substitution to improve reproducibility should be updated once 10.0.0 is
# released. See the comment in `patchPhase` for more information.
```

Otherwise, this will most likely be missed in a future update.

> Note: noticed the `/etc/vim` replacement doesn't do anything.

Ah, indeed. Could you remove it in a separate commit in this PR?

Reply:
as requested...